Next

pet1wz opened the pull request.

Qwebeck committed:
Task 438 new app structure (#340)

Dimonium-239 committed:
TASK-400 Export and refresh DB from modal (#320)

kaplonpaulina committed:
TASK-418 Disallow working shifts that cross 7am  (#341)

* add check

* handle non working shifts

* Change function names and moves 7 to const

Co-authored-by: Bohdan Forostianyi <qwebeck5@gmail.com>

pet1wz committed:
Task-455 Css refactor buttons dropdown (#343)

net-runner committed:
TASK-107 app localization (#355)

:)

przemec committed:
TASK-451 App version fetched from GitHub (#357)

kopcion committed:
TASK-444 test refactor part 2, from origin/next (#360)

pet1wz committed:
TASK-442 Introduce WTC errors  (#359)

Sig00rd committed:
TASK-450 Algorithm errors found info (#361)

* Add view and logic for info about algorithm errors found

* Make use of translations for the error text, remove redundant useState typings

przemec committed:
Task 421 new version modal (#363)

pet1wz committed:
TASK-459 Styled components migration (#362)

Qwebeck committed:
Task 437 typescript migration (#364)

Qwebeck reviewed (changes requested):
Nie mergować zanim będzie zrobione to zadanie 
https://app.tara.ai/nurse-scheduling/team-nurse/tasks/478

przemec committed:
TASK-482 (#366)

Ehevi committed:
Task 307 Add holidays information in request to the server (#367)

Qwebeck committed:
Added pre-push config (#365)

Qwebeck committed:
Removed unused code (#372)

Qwebeck committed:
Split shift cell on smaller parts (#374)

przemec committed:
TASK-356 'come back to now' button work test (#371)

net-runner committed:
TASK-153 Change worker hours e2e test. (#368)

* PR ready.

* Adressed pr changes.

* Adressed PR changes.

* Adressed PR review changes.

* Fixes for tests.

* Removed obsolete test.

Co-authored-by: Tomasz Pęcak <pecatoma@gmail.com>

Ehevi committed:
TASK-449 Export of monthly leaves (#369)

net-runner committed:
TASK-354 not saved schedule changes are not displayed e2e test (#373)

Qwebeck committed:
Task 466 implementing redux best practices (#381)

pet1wz committed:
TASK-497 Setup SonarQube (#383)

* Setup sonar qube job

* Add action for next PR's

pet1wz committed:
Change sonar project key (#384)

pet1wz committed:
Sonarqube setup (#387)

* Change sonar project key

* Add sonar cloud

net-runner committed:
TASK-358 change workers shift type e2e test (#377)

* PR ready.

* Merge conflicts resolved.

* Adressed pr review changes.

* Refactored Route Buttons, so they should render faster and test should now pass

Co-authored-by: Bohdan Forostianyi <qwebeck5@gmail.com>

kopcion committed:
TASK-172  Add test badge (#370)

* .

* TASK-172 adding test coverage badge attempt no.1

* TASK-172 adding test coverage badge attempt no.2

* TASK-172 adding test coverage badge attempt no.3

* TASK-172 adding test coverage badge attempt no.4

* TASK-172 adding test coverage badge attempt no.5

* TASK-172 adding test coverage badge attempt no.6

* TASK-172 adding test coverage badge attempt no.7

* TASK-172 adding test coverage badge attempt no.8

* TASK-172 adding test coverage badge attempt no.9

* TASK-172 adding test coverage badge attempt no.10

* TASK-172 adding test coverage badge attempt no.11

* TASK-172 adding test coverage badge attempt no.12

* TASK-172 adding test coverage badge attempt no.13

* TASK-172 adding test coverage badge attempt no.13

* TASK-172 adding test coverage badge attempt no.13

* TASK-172 adding test coverage badge attempt no.15

* TASK-172 adding test coverage badge attempt no.15

* TASK-172 adding test coverage badge attempt no.17

* TASK-172 adding test coverage badge attempt no.17

* TASK-172 adding test coverage badge attempt no.19

* TASK-172 adding test coverage badge attempt no.20

* TASK-172 adding test coverage badge attempt no.21

* TASK-172 adding test coverage badge attempt no.22

* TASK-172 adding test coverage badge attempt no.24

* TASK-172 adding test coverage badge attempt no.24

* TASK-172 adding test coverage badge attempt no.26

* TASK-172 adding test coverage badge attempt no.27

* TASK-172 adding test coverage badge attempt no.28

* TASK-172 adding test coverage badge attempt no.28

* TASK-172 adding test coverage badge attempt no.30

* TASK-172 adding test coverage badge attempt no.31

* TASK-172 adding test coverage badge attempt no.31

* TASK-172 adding test coverage badge attempt no.33

* TASK-172 adding test coverage badge attempt no.34

* TASK-172 adding test coverage badge attempt no.35

* TASK-172 adding test coverage badge attempt no.36

* TASK-172 adding test coverage badge attempt no.37

* TASK-172 adding test coverage badge attempt no.38

* TASK-172 adding test coverage badge attempt no.38

* TASK-172 adding test coverage badge attempt no.39

* TASK-172 adding test coverage badge attempt no.40

* TASK-172 adding test coverage badge attempt no.41

* TASK-172 review fix

* Update README.md

Co-authored-by: Mateusz Kopcinski <mateusz.kopcisnki@swmansion.com>
Co-authored-by: Paweł Renc <rencpawe@gmail.com>

net-runner committed:
TASK-498 Fix Sonarqube detected problems (#390)

pet1wz committed:
Task 465 refactor database interface (#380)

Qwebeck committed:
Task 474 todo walkthrough+task 467 useless components removal (#392)

kopcion committed:
Update README.md

Fixed coverage badge

przemec committed:
TASK-473 styles migration to styled components (#378)

przemec committed:
error modal style fixed + useless code cleanup (#394)

Qwebeck committed:
Fixes sonar qube errors (#397)

Qwebeck committed:
Fixed problem with bad hour calculation (#395)

Qwebeck committed:
Fixes bad export of worker hours (#396)

net-runner committed:
TASK-475 & 472 styledcomponents theme implementation and css style reset without bootstrap scss (#398)

* Implemented theming system

* Reset styles without scss

* Added Roboto font to project

* Removed obsolete node-sass & fontsource-roboto modules.

* Adds chokidar as dependency

Co-authored-by: Bohdan Forostianyi <qwebeck5@gmail.com>

Qwebeck committed:
Fixed bug where application state revert where not possible (#400)

Qwebeck committed:
Fixed error in name to uuid mapping (#399)

venglov committed:
TASK-427 Export single worker calendar to xlsx (#388)

net-runner committed:
TASK-515 new schedule design implementation (#401)

pet1wz committed:
Rework package.json scripts (#405)

Qwebeck committed:
Added firebase (#403)

Qwebeck committed:
Next pwa firebase sync (#406)

* Initial + redux-persist + firebase + tauri.

* Added simple login.

* modified:   README.md
	modified:   package-lock.json
	modified:   package.json
	deleted:    public/electron.js
	modified:   public/index.html
	modified:   public/manifest.json
	modified:   src/index.tsx
	new file:   src/service-worker.ts
	modified:   src/serviceWorker.ts

* modified:   public/index.html

* 💯

* PWAifyied 👽

* Killed husky. Bad dog 🐕

* updated style of management page

* updated Caretaker icon svg

* added svg icons to WorkerType spans

* updated Nurse icon svg

* Firebase sync in progress.

* updated part of firebase persist provider

* Firebase syncd 🔄 🤨

* fixed schedule page style

* added option to drawer to be full viewport height

* added text overflow ellipsis to names in schedule table

* fixed styles of worker edit drawer

* Load/Save schedule if exists onLogin.

* Added localization for schedule downloading.

* Added missing translation link.

Co-authored-by: net-runner <slazynski.j@gmail.com>
Co-authored-by: Przemec <pszczepaniak000@gmail.com>

Qwebeck committed:
Revert "Next pwa firebase sync (#406)" (#407)

This reverts commit dea2cd81112535dccc8c4c9f5f9ee1c19cfdb207.

Qwebeck committed:
Next pwa firebase sync (#415)

* Initial + redux-persist + firebase + tauri.

* Added simple login.

* modified:   README.md
	modified:   package-lock.json
	modified:   package.json
	deleted:    public/electron.js
	modified:   public/index.html
	modified:   public/manifest.json
	modified:   src/index.tsx
	new file:   src/service-worker.ts
	modified:   src/serviceWorker.ts

* modified:   public/index.html

* 💯

* PWAifyied 👽

* Killed husky. Bad dog 🐕

* updated style of management page

* updated Caretaker icon svg

* added svg icons to WorkerType spans

* updated Nurse icon svg

* Firebase sync in progress.

* updated part of firebase persist provider

* Firebase syncd 🔄 🤨

* fixed schedule page style

* added option to drawer to be full viewport height

* added text overflow ellipsis to names in schedule table

* fixed styles of worker edit drawer

* Load/Save schedule if exists onLogin.

* Added localization for schedule downloading.

* Added missing translation link.

* Added typings for store, state helper hooks and fixed fb connection.

* Removed unused links.

Co-authored-by: net-runner <slazynski.j@gmail.com>
Co-authored-by: Przemec <pszczepaniak000@gmail.com>

net-runner committed:
closes 414 remove buttons for edition from shift management tab (#416)

* Initial + redux-persist + firebase + tauri.

* Added simple login.

* modified:   README.md
	modified:   package-lock.json
	modified:   package.json
	deleted:    public/electron.js
	modified:   public/index.html
	modified:   public/manifest.json
	modified:   src/index.tsx
	new file:   src/service-worker.ts
	modified:   src/serviceWorker.ts

* modified:   public/index.html

* 💯

* PWAifyied 👽

* Killed husky. Bad dog 🐕

* updated style of management page

* updated Caretaker icon svg

* added svg icons to WorkerType spans

* updated Nurse icon svg

* Firebase sync in progress.

* updated part of firebase persist provider

* Firebase syncd 🔄 🤨

* fixed schedule page style

* added option to drawer to be full viewport height

* added text overflow ellipsis to names in schedule table

* fixed styles of worker edit drawer

* Load/Save schedule if exists onLogin.

* Added localization for schedule downloading.

* Added missing translation link.

* Added typings for store, state helper hooks and fixed fb connection.

* Removed unused links.

* Buttons removed. Issue finished.

* ... 😠

* .... sc 👍

Co-authored-by: Przemec <pszczepaniak000@gmail.com>

Qwebeck committed:
408+413 (#419)

* Worker edit drawer styled. Removed tauri folder.

* Styled revision switch.

* Styled file button.

* Created reusable worker label component.

* New style for worker info table.

* Small table height fix.

* Styling for worker drawer info finished.

* PL translations for worker drawer info changed to lowercase.

* Removed warnings

Co-authored-by: net-runner <slazynski.j@gmail.com>

net-runner committed:
Error modal restyle finished. (#423)

net-runner committed:
Now dropdown button group has a visible feedback on hover. (#424)

Qwebeck committed:
#431: disables shift editing in read mode (#432)

Qwebeck committed:
#410: remove area selection functionality (#430)

Qwebeck committed:
Updates react and ts versions (#436)

* Updates react and ts versions

* fix: Use yarn as a default and only package manager.

Co-authored-by: net-runner <slazynski.j@gmail.com>

net-runner committed:
429 Change Glider Logo (#437)

* Updates react and ts versions

* fix: Use yarn as a default and only package manager.

* feat: New Glider Logo.

Co-authored-by: Bohdan <qwebeck5@gmail.com>
